### ISSUE TITLE
[INFRA-249] Kohsuke might be the only one who can decrypt statistics data

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkins-infra/helpdesk#240]
---
<details><summary><i>Originally reported by <a title="R. Tyler Croy" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=rtyler">rtyler</a>, imported from: <a href="https://issues.jenkins.io/browse/INFRA-249" target="_blank">Kohsuke might be the only one who can decrypt statistics data</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Kohsuke Kawaguchi" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kohsuke">kohsuke</a>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>imported</b>: 2022/01/10
</ul></i>
</details>

**Comment by timja** [comment]
@abayer manages this now afaik, there's other tickets for automating via github actions

**Comment by abayer** [comment]
I've always handled the usage stats site generation, but KK still owns the tooling/automation that takes the raw usage logs, decrypts them, scrubs them of identifying information, and pushes them back up to usage.jenkins.io. That said, I believe at least @olblak has the relevant key now?

**Comment by olblak** [comment]
> 

Yup, If I remember correctly I have the key which I should transfer to @dduportal

**Comment by dduportal** [comment]
Reopening as a sub item of #4512
